### ISSUE TITLE
TouchUI Console for ACS Commons (replace /miscadmin)

### ISSUE THREAD

**Issue by davidjgonzalez** [Adobe-Consulting-Services/acs-aem-commons#2354]
It would be nice to have a TouchUI console that has the same core features of the ClassicUI `/miscadmin` console.

* Create new page
* Copy
* Move 
* Delete
* Lock/Unlock
* Publish/Unpublish

![image](https://user-images.githubusercontent.com/1451868/86371401-d679d100-bc4e-11ea-94ee-78137e092608.png)

**Comment by kwin** [comment]
Why not using Context-Aware Configurations with https://wcm.io/caconfig/editor/?
As "/etc" is going away anyways, these pages need to be moved to another location anyways...

**Comment by davidjgonzalez** [comment]
@kwin most of the ACS here aren't configurations (rather content or utility pages)

Relying on wcm.io (which I very much wcm.io projects, don't get me wrong), violates the ACS Commons tenant of not requiring 3rd party dependencies in a big way; so their CAConfig editor wouldn't work.

Instead of moving to `/conf` (i have a fundamental irritation with how `/conf` is generally being (IMO) abused (even by the product) and turning into another `/etc`) -- I'd much rather see these moved to `/content/acs-commons` with a clean way to add `/content/acs-commons` to the OOTB Siteadmin exclude list (as noted in original issue). 

One nice thing about this approach is it is effectively, non-breaking parity w/ the current miscadmin-based ACS Commons functionality; so we aren't creating more work for ourselves in the future if/when we move away from `/etc/acs-commons` .. all this does is help us work in a fully TouchUI world (understanding we need to move the pages themselves off classic UI, but we have to do that regardless)

**Comment by davidjgonzalez** [comment]
@justinedelson, do you have any insight you can share into the short/medium/long term realities of "/etc going away"? and/or what it's replacement will be (a mix of /conf and /content, or a new root, or a continuation of using /etc for specific use cases not satisfied by /conf or /content)

**Comment by davidjgonzalez** [comment]
Though, this appears to be a problem w/ this impl:

```
[INFO] --- oakpal-maven-plugin:2.0.0:verify (oakpal-verify) @ acs-aem-commons-ui.apps ---
[INFO] OakPAL Check Reports
[INFO]   com.adobe.acs.acs-aem-commons-oakpal-checks/content-class-aem64/content-classifications
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/activities [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/annotations [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/comments [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/default [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/versions [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/workflows [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[INFO]   com.adobe.acs.acs-aem-commons-oakpal-checks/content-class-aem65/content-classifications
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/activities [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/annotations [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/comments [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/default [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/versions [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR]    +- <MAJOR> /apps/acs-commons/console/jcr:content/rails/timeline/items/timeline/items/filter/items/workflows [restricted resource type]: /libs/granite/ui/components/foundation/form/option is marked INTERNAL [acs-aem-commons-ui.apps-4.7.1-SNAPSHOT.zip]
[ERROR] ** Violations were reported at or above severity: MAJOR **
```

**Comment by badvision** [comment]
@davidjgonzalez can we close this issue now or is there something not resolved about it still?

**Comment by davidjgonzalez** [comment]
Yup - it looks like it was merged/released with 4.7.2 (not sure we can update the milestone - but either way, we can close)
